package.json

```diff
@@ -79,6 +79,11 @@
           },
           "description": "Arguments for clangd server"
         },
+        "clangd.semanticHighlighting": {
+          "type": "boolean",
+          "default": "true",
```

Comment:
can we default this to false and try it out for a while?
I'm a little worried about performance, particularly in vim where I've had problems with update performance in highlighting plugins before.